apps/src/lib/ui/SettingsCog.jsx

```diff
@@ -103,6 +102,11 @@ class SettingsCog extends Component {
     return pageConstants && pageConstants.aiEnabled;
   }
 
+  isProjectTemplateLevel() {
```

Comment:
Looks good! Only comment here is that I wonder if there is a way to de-duplicate this repeated `let pageConstants = getStore().getState().pageConstants;
    return pageConstants && pageConstants.<property>;`. Not sure what that might be off the top of my head, but happy to brainstorm on possibilities!

Reply:
Just added a function called `getPageConstants`. Thanks for the suggestion!